site/source/tutorials/001_static_content/index.md

```diff
@@ -14,17 +16,29 @@ You can [download](../assets/001_static_content-initial.zip) the demo project to
 
 You also need to be familiar with TypeScript as Dojo 2 uses it extensively. For more information, refer to the [TypeScript and Dojo 2](../comingsoon.html) article.
 
-## Printing static content
+{% section %}
+
+## Starting the development server
 
-Now it is time to start customizing the application. To begin, let's startup the application with the development server and observe the impact of our changes. Run the following command in the application's root directory:
+{% task 'Build and run the application' %}
+
+Before we start making changes, let's start the application with the development server so that we can observe the impact of our changes. Run the following command in the application's root directory:
 
 `dojo build -w`
 
 Now, open up a web browser to [http://localhost:9999](http://localhost:9999) to view the current application.
 
-To start customizing the application, let's remove the content that is already there. There are two places we need to go to do this.
+In the next step, we will start to customize the application.
+
+{% section %}
+
+## Page content
+
+{% task 'Change what is rendered to the page' %}
+
+To start customizing the application, let's remove the content that is already there. There are two places we need to go to do this. The first line, "Welcome to biz-e-corp" is being generated from the `index.html` file.
```

Comment:
is being generated from the `index.html` file. -> is being generated from `index.html`.